plugin/evm/mempool.go

```diff
@@ -8,10 +8,13 @@ import (
 	"fmt"
 	"sync"
 
+	"github.com/ethereum/go-ethereum/log"
+
 	"github.com/ava-labs/avalanchego/cache"
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/network/p2p/gossip"
+
 	"github.com/ava-labs/coreth/metrics"
-	"github.com/ethereum/go-ethereum/log"
 )
 
 const (
```

Comment:
Move `discardedTxsCacheSize` to `vm.go`?

Reply:
I think this is unrelated to the current diff (correct me if wrong) so I'm down to do this as followup instead